Leanplum-SDK/Classes/LPDatabase.m

```diff
@@ -49,8 +49,17 @@ - (id)init
  */
 - (sqlite3 *)initSQLite
 {
+    if (sqlite3_threadsafe() != SQLITE_CONFIG_SERIALIZED) {
```

Comment:
As discussed offline, let's should remove this block.

Note from https://sqlite.org/c3ref/initialize.html
"The sqlite3_initialize() interface is threadsafe, but sqlite3_shutdown() is not. The sqlite3_shutdown() interface must only be called from a single thread. All open database connections must be closed and all other SQLite resources must be deallocated prior to invoking sqlite3_shutdown()."

There are no guarantee that other SDK won't call this.

Reply:
yep. removed it. good catch too :)